### ISSUE TITLE
Expose non-`lens` accessors

### ISSUE THREAD

**Issue by elldritch** [brendanhay/semver#12]
I'm working on a team whose application that intentionally does not use lenses (for onboarding purposes). Unfortunately, this prevents me from accessing fields of `Version` without cobbling together some ad-hoc lenses.

I wish this package provided accessors that were not lens-style e.g. `release' :: Version -> [Identifier]`.

**Comment by elldritch** [comment]
It looks like HEAD on `main` already exposes the `Data.SemVer.Internal` module, which has what I need. Can you publish a new version with the current changes?

**Comment by ProofOfKeags** [comment]
I don't believe I have access to be able to push a new version to Hackage, but @brendanhay might.
